### PR TITLE
KT-21213 multiline kdoc - intellij joins lines together without space

### PR DESCRIPTION
**Pull request by t-kameyama**
https://youtrack.jetbrains.com/issue/KT-21213

### CODE REVIEW

File: idea/src/org/jetbrains/kotlin/idea/kdoc/KDocRenderer.kt [hunk 1]
```diff
@@ -163,7 +164,10 @@ object KDocRenderer {
         // Avoid wrapping the entire converted contents in a <p> tag if it's just a single paragraph
         val maybeSingleParagraph = markdownNode.children.singleOrNull { it.type != MarkdownTokenTypes.EOL }
         return if (maybeSingleParagraph != null && !allowSingleParagraph) {
-            maybeSingleParagraph.children.joinToString("") { it.toHtml() }
+            val lineSeparator = LineSeparator.getSystemLineSeparator().separatorString
```
Comment (yole) [inline idea/src/org/jetbrains/kotlin/idea/kdoc/KDocRenderer.kt]: Source code in IntelliJ always uses `\n` as a line separator. No need to access the system line separator here.

**Comment by t-kameyama** [comment]
Fixed. Please review.

**Comment by yole** [comment]
You can also remove the unused import.

**Comment by t-kameyama** [comment]
Removed it.

**Pull request merged**